app/views/availabilities/index.html.erb

```diff
@@ -21,9 +21,10 @@
           <% start_time += 1.hour %>
         <% end %>
       </tr>
-      <% Availability.days.each_key do |day| %>
+      <% (1..7).each do |index| %>
+        <% day = Availability.days.keys[index % 7] %>
         <tr>
-          <td class="dayname"><%= day %></td>
+          <td class="dayname"><%= day[0..2] %></td>
```

Comment:
Still can't do anything about this huh... Okay just ignore it

Reply:
We decided to standardise all day on Date::DAYNAMES since other solution are equally hacky.